### PR TITLE
greetings

### PR DESCRIPTION
**Pull request by tovernaar123**
greets player on login

### CODE REVIEW

File: modules/addons/greetings.lua [hunk 1]
```diff
@@ -0,0 +1,14 @@
+--greets players on join
+
+local Event = require 'utils.event' --- @dep utils.event
+local greet = 
+function(event) 
+    local message = {'greetings.greet',{'links.discord'}}
+    game.players[event.player_index].print(message)
+
+
```
Comment (bbassie) [inline modules/addons/greetings.lua]: Try to avoid whitespace like this

File: modules/addons/greetings.lua [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+end
+
+
+
```
Comment (bbassie) [inline modules/addons/greetings.lua]: Try to avoid whitespace like this

File: locale/en/addons.cfg [hunk 3]
```diff
@@ -57,3 +57,8 @@ riot=(admins) ┬┴┬┴┤ᵒ_ᵒ)├┬┴┬┴ ‹ ‹\(´ω` )/››‹
 loops=NO LOOPS; LOOPS ARE BAD; JUST NO LOOPS!!!!!; IF YOU MAKE A LOOP.... IT WILL NOT END WELL!!!!!!!
 lenny=( ͡° ͜ʖ ͡°)
 hodor=Hodor
+
+
+
```
Comment (bbassie) [inline locale/en/addons.cfg]: Remove this white space to keep consistent with the rest of the file

Comment (tovernaar123) [reply locale/en/addons.cfg]: Sorry i made it in 5 mins with the auto station naming, next time i make a pr i will remove the white space to.

**Pull request merged**